Assets/Live2D/Cubism/Framework/LookAt/CubismLookParameter.cs

```diff
@@ -75,7 +75,7 @@ private void Reset()
         /// </summary>
         /// <param name="targetOffset">Delta to target.</param>
         /// <returns>Evaluation result.</returns>
-        internal float TickAndEvaluate(Vector3 targetOffset)
+        public float TickAndEvaluate(Vector3 targetOffset)
```

Comment:
Typically, this function is accessed from CubismLookController, so there is no need to make it public. Could you please let us know the intended use case for this?